public/app/features/plugins/components/PluginsErrorsInfo.tsx

```diff
@@ -1,42 +1,19 @@
 import React from 'react';
 import { selectors } from '@grafana/e2e-selectors';
 import { HorizontalGroup, InfoBox, List, PluginSignatureBadge, useTheme } from '@grafana/ui';
-import { StoreState } from '../../../types';
-import { getAllPluginsErrors } from '../state/selectors';
-import { loadPlugins, loadPluginsErrors } from '../state/actions';
-import useAsync from 'react-use/lib/useAsync';
-import { connect, ConnectedProps } from 'react-redux';
+import { useGetErrors, useFetchStatus } from '../admin/state/hooks';
 import { PluginErrorCode, PluginSignatureStatus } from '@grafana/data';
 import { css } from '@emotion/css';
 
-const mapStateToProps = (state: StoreState) => ({
-  errors: getAllPluginsErrors(state.plugins),
-});
-
-const mapDispatchToProps = {
-  loadPluginsErrors,
-};
-
-interface OwnProps {
-  children?: React.ReactNode;
-}
-const connector = connect(mapStateToProps, mapDispatchToProps);
-type PluginsErrorsInfoProps = ConnectedProps<typeof connector> & OwnProps;
-
-export const PluginsErrorsInfoUnconnected: React.FC<PluginsErrorsInfoProps> = ({
-  loadPluginsErrors,
-  errors,
-  children,
-}) => {
+export const PluginsErrorsInfo: React.FC = ({ children }) => {
```

Comment:
nit: I think we prefer functional components to be defined like the following these days:

```suggestion
export function PluginsErrorsInfo({ children }): React.ReactElement | null {
```

Reply:
Absolutely, will change 👍

Reply:
Changed in https://github.com/grafana/grafana/pull/41739/commits/ff287df5b61fe0d7eb7e20380a77af0a3791f7ab.